Common/UnitDefinitions/SpecificFuelConsumption.json

```diff
@@ -31,6 +36,10 @@
     {
       "SingularName": "GramPerKiloNewtonSecond",
       "PluralName": "GramsPerKiloNewtonSecond",
+      "BaseUnits": {
+        "L": "Meter",
+        "T": "Microsecond"
```

Comment:
Meter, Second?

Not entirely sure how to correctly resolve the SI base units for this.
I tried wolfram, but no clear cut answer.

https://www.wolframalpha.com/input?i=Gram+Per+KiloNewton+Second
![image](https://github.com/user-attachments/assets/a00f5c1a-3f06-416c-9d53-ea9fef05274c)

Reply:
I think the official SI unit here would be `GramPerNewtonSecond` - the `Mass` dimension is being simplified by the `Force`. Since we're left with a `1/Kilo` coefficient (and I didn't want to leave it undefined) - I've used the `Kilo` prefix of the `Time` dimension (I could have also picked the `LengthUnit` to scale on, but with an exponent of `"L": -1` this becomes a little harder to parse).

This is good example of the _choice_ I had to make in the (yet unpublished) fix for the generation of the prefix-bases - with more than one dimensions, there are often multiple possible combinations of prefixes to choose from, so I decided on the following order: "1, -1, 2, -2, ... n, -n" where the number represents a given dimension (e..g. if I have `"L" : 2, "T": -1` i would scale (if possible) using the `T` (as the scaling factor represents a _smaller deviation_).

Reply:
I still don't 100% follow here, but it kind of maybe makes sense. I'm lacking some basic understanding of how to think of this.

So, this is my first, uneducated attempt at breaking this down:

```
1 N = 1 kg⋅m⋅s −2

GramsPerKiloNewtonSecond =>
1g/(1000N * s) => 
1g/(1000kg⋅m⋅s−2 * s) => 
1g/(1000kg⋅m⋅s−1) =>
(1g * s) / (1000kg⋅m) =>
(1000 * 1g * s) / (kg⋅m) =>
(1kg * s) / (kg⋅m) =>
1s / m
```

So.. I'm probably way off here, but I arrive at 1 second per meter in SI base units.
Is this the way to go about it?

Reply:
> I still don't 100% follow here, but it kind of maybe makes sense. I'm lacking some basic understanding of how to think of this.
> 
> So, this is my first, uneducated attempt at breaking this down:
> 
> ```
> 1 N = 1 kg⋅m⋅s −2
> 
> GramsPerKiloNewtonSecond =>
> 1g/(1000N * s) => 
> 1g/(1000kg⋅m⋅s−2 * s) => 
> 1g/(1000kg⋅m⋅s−1) =>
> (1g * s) / (1000kg⋅m) =>
> (1000 * 1g * s) / (kg⋅m) =>
> (1kg * s) / (kg⋅m) =>
> 1s / m
> ```
> 
> So.. I'm probably way off here, but I arrive at 1 second per meter in SI base units. Is this the way to go about it?

Yes, you're totally right- it's always the `Kilogram` that messes up my Si unit estimations...

Reply:
Fixed in https://github.com/angularsen/UnitsNet/pull/1473/commits/c32d2ffd787d5a039007aab8ee19af7f10a8bd89

Reply:
Ok nice, I learned something new today then 🎉